pkgs/development/python-modules/grad-cam/default.nix

```diff
@@ -17,24 +18,31 @@
 
 buildPythonPackage rec {
   pname = "grad-cam";
-  version = "1.5.0";
+  version = "1.5.2";
   pyproject = true;
 
   disabled = pythonOlder "3.8";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-aw7Z/6/AMKH2PVBcOr8HxsmRDa6c3v8Xd4xa8HTiFGA=";
+    hash = "sha256-WhC3QjMDh4E8NnO8SyCtg9rFgDJkYP6/xdWNcTvlKFU=";
   };
 
   postPatch = ''
     substituteInPlace requirements.txt\
       --replace "opencv-python" "opencv"
   '';
 
-  nativeBuildInputs = [ setuptools ];
+  build-system = [
+    pythonRelaxDepsHook
```

Comment:
`pythonRelaxDepsHook` should be kept in `nativeBuildInputs`.

Reply:
Done